com.civicrm.hrjobroles/js/test/controllers/hr-job-roles-controller/hr-job-roles-controller_test.js

```diff
@@ -27,6 +27,91 @@ define([
             DateValidation.dateFormats.push('DD/MM/YYYY');
         }));
 
+        describe('getOptionValues', function() {
+
+          it('should build the "CostCentreList" array of objects containing the "weight" property', function(){
+            spyOn(HRJobRolesService, 'getOptionValues').and.returnValue($q.resolve({
```

Comment:
The `spyOn`, `initController()`, and `$rootScope.$digest()` are better placed in a `beforeEach()` block rather than in the `it` block where, as much as possible, only the `expect()` statements should be placed